src/Paprika/Store/AbandonedPage.cs

```diff
@@ -24,7 +24,7 @@ namespace Paprika.Store;
 
     private unsafe ref Payload Data => ref Unsafe.AsRef<Payload>(page.Payload);
 
-    [StructLayout(LayoutKind.Explicit, Size = Size)]
+    [StructLayout(LayoutKind.Sequential, Pack = sizeof(byte), Size = Size)]
```

Comment:
This is also persistent thing. Cannot go sequential I believe.

Reply:
Sequential is fine, just don't move fields 🙂